spa/.yarnrc

```diff
@@ -0,0 +1,2 @@
+--install.ignore-engines true
```

Comment:
One of the sub deps in  latest react-scripts requires node16. 
We are using node14.
So ignore it for now, will upgrade later.